builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -218,101 +218,6 @@ node()  {
       error "service platform is unknown: ${servicePlatform}"
     }
 
-		stage("Deployment to ${current_environment} Env"){
```

Comment:
Is this deployment stage not needed?

Reply:
It is needed and it right there in the code. It was again duplicated because of merge conflict